src/content/contributing/index.md

```diff
@@ -1,12 +1,313 @@
 ---
 title: Contributing
 lang: en
+sidebar: true
 ---
 
-# Contributing to ethereum.org
+# Contributing to ethereum.org 🦄
 
-This page is coming soon!
+ethereum.org is an open-source project. So if you want to help improve our portal to Ethereum, here's how you can help out.
 
-In the meantime, [learn how to contribute](https://github.com/ethereum/ethereum-org-website#how-can-i-contribute).
+## Ways to contribute
 
-Looking to add a dapp or wallet to the site? Learn [how to add a product](/en/contributing/adding-products/).
+- [Work on an open issue](https://github.com/ethereum/ethereum-org-website/issues)
+  _– Work we've identified that needs doing_
+- [Join the translations program ](https://ethereum.org/en/languages/#ethereum-org-translation-program)
```

Comment:
```suggestion
- [Join the translations program ](/en/languages/#ethereum-org-translation-program)
```
Internal links shouldn't mention "ethereum.org" - otherwise they will open in a new tab (unless that's the intention here).